docs-src/typescript/how-to-use-environment-variables-in-typescript.md

```diff
@@ -88,3 +88,5 @@ async function yourWorkflow() {
   await sendNotificationEmail(envVars.apiKey);
 }
 ```
+
+If your environment variables are sensitive, use a [Custom Data Converter that encrypts](https://legacy-documentation-sdks.temporal.io/typescript/data-converters#encryption) Activity return values and parameters.
```

Comment:
Will likely link to `concepts/what-is-a-custom-data-converter.md` after [PR 1932](https://github.com/temporalio/documentation/pull/1932) is merged.
```suggestion
If your environment variables are sensitive, use a [custom Data Converter](https://legacy-documentation-sdks.temporal.io/typescript/data-converters#encryption) to encrypt Activity return values and parameters.
```